index.html

```diff
@@ -65,190 +70,239 @@ <h2 class="icon-text">LET'S CONNECT</h2>
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+            <button type="button" class="project-btn">see project</button>
           </div>
-          <p class="project-paragraph">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul class="technologies">
-            <li>html</li>
-            <li>css</li>
-            <li>javaScript</li>
-          </ul>
-          <button type="button" class="project-btn">see project</button>
         </article>
         <!--fourth card-->
-        <article class="project-card">
+        <article class="project-card reverse">
           <div class="project-img">
             <img
-              src="Images/Fourth.png"
+              src="Images/test4.svg"
               alt="a brief description of what we do"
             />
           </div>
-          <h2 class="project-title">Multi-post stories</h2>
-          <div class="info">
-            <span class="canopy !important">CANOPY</span>
-            <img src="./Images/Counter.png" alt="dot" />
-            <span class="">back end dev</span>
-            <img src="./Images/Counter.png" alt="dot" />
-            <span>2015</span>
+          <div class="card-info">
+            <h2 class="project-title">Multi-Post Stories</h2>
+            <div class="info">
+              <span class="canopy">CANOPY</span>
+              <img src="./Images/Counter.png" alt="dot" />
+              <span class="">back end dev</span>
+              <img src="./Images/Counter.png" alt="dot" />
+              <span>2015</span>
+            </div>
+            <p class="project-paragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="technologies">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+            <button type="button" class="project-btn">see project</button>
           </div>
-          <p class="project-paragraph">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul class="technologies">
-            <li>html</li>
-            <li>css</li>
-            <li>javaScript</li>
-          </ul>
-          <button type="button" class="project-btn">see project</button>
         </article>
       </section>
       <section class="about">
         <div>
-          <h1 class="intro-text down">About Myself</h1>
-          <p class="text-2">
-            Hello I’m a software developer! I can help you build a product ,
-            feature or website Look through some of my work and experience! If
-            you like what you see and have a project you need coded, don’t
-            hestiate to contact me.
-          </p>
+          <div>
```

Comment:
- [ ] KIndly follow the design template used for the about section. I will like to recommend that you stick to the CSS declarations used for this section. You can use `display: grid` or `display: flex` to style this section and make it responsive 🙏 

## Your Design

![image](https://user-images.githubusercontent.com/77106962/144683658-f49ed673-78ff-42ca-8d50-8d6ce9186cdf.png)


## Figma's Design

![image](https://user-images.githubusercontent.com/77106962/144683728-d188f461-193d-4081-bdbd-f34417fa3907.png)


### Kindly implement the changes 👍